src/sentry/monitors/models.py

```diff
@@ -292,6 +292,9 @@ def get_next_scheduled_checkin_with_margin(self, last_checkin):
         next_checkin = self.get_next_scheduled_checkin(last_checkin)
         return next_checkin + timedelta(minutes=int(self.config.get("checkin_margin") or 0))
 
+    def get_checkin_with_margin(self, checkin):
+        return checkin + timedelta(minutes=int(self.config.get("checkin_margin") or 0))
```

Comment:
Should we just call this 
```suggestion
    def add_checkin_margin(self, checkin_time: datetime):
        return checkin_time + timedelta(minutes=int(self.config.get("checkin_margin") or 0))
```

Reply:
sure yeah. it's kind of an annoying method to have but i didn't want to pull the value out of the config twice